app.py

```diff
@@ -46,6 +47,42 @@ def scrape_airbnb_reviews(url):
     finally:
         driver.quit()
 
+def scrape_airbnb_amenities(url):
+    driver = connect_browser()
+    
+    try:
+        driver.get(url)
+        WebDriverWait(driver, 60).until(EC.presence_of_element_located((By.XPATH, "//button[contains(text(), 'Show all 41 amenities')]")))
+        show_all_amenities_button = WebDriverWait(driver, 10).until(
+            EC.element_to_be_clickable((By.XPATH, "//button[contains(text(), 'Show all 41 amenities')]"))
+        )
+        show_all_amenities_button.click()
+
+        WebDriverWait(driver, 20).until(
+            EC.presence_of_element_located((By.CSS_SELECTOR, '[data-testid="modal-container"]'))
+        )
+        
+        amenities_elements = driver.find_elements(By.CSS_SELECTOR, "._11jhslp")
+        amenities = []
+        
+        for element in amenities_elements:
+            name = element.find_element(By.CSS_SELECTOR, 'h3').text.strip()
+            content_amenities_element = element.find_elements(By.CSS_SELECTOR, 'li')
+            services = []
+            for item in content_amenities_element:
+                service_text = item.find_element(By.XPATH, ".//div[contains(@class, 'twad414')]").text
```

Comment:
hmmm.. these classes might not work. can u test on different properties

Reply:
Ok I'll try that

Reply:
It works for now also i cannot find a more generalised way to extract the amenities other than this one